guides/guide-for-adding-windows-node.md

```diff
@@ -1,229 +1,402 @@
 
 # Adding Windows nodes
 
-  
-
 You can use Kubernetes to run a mixture of Linux and Windows nodes, so you can mix Pods that run on Linux on with Pods that run on Windows. This is a guide on how to register Windows nodes to your cluster.
 
-  
 
 ## Before you begin
 
-  
-
 Your Kubernetes server must be at or later than version 1.22. To check the version, enter `kubectl version`.
 
-  
-
 - Obtain a [Windows Server 2019 license](https://www.microsoft.com/en-us/cloud-platform/windows-server-pricing) (or higher) in order to configure the Windows node that hosts Windows containers. If you are using VXLAN/Overlay networking you must have also have [KB4489899](https://support.microsoft.com/help/4489899) installed.
 
 - A Linux-based Kubernetes kubeadm cluster in which you have access to the control plane (see [Creating a single control-plane cluster with kubeadm](https://kubernetes-docsy-staging.netlify.app/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/)).
 
-  
 
 ## Objectives
 
-  
-
 - Register a Windows node to the cluster
-
 - Configure networking so Pods and Services on Linux and Windows can communicate with each other
 
-  
 
 ## Getting Started: Adding a Windows Node to Your Cluster
 
-  
 
 ### Networking Configuration
 
-  
-
 Once you have a Linux-based Kubernetes control-plane node you are ready to choose a networking solution.
 
-  
 
-#### Configuring Flannel with rancher
-
-  
+## Configuring Flannel with rancher
 
 1. Prepare Kubernetes control plane for Flannel
 
 Some minor preparation is recommended on the Kubernetes control plane in our cluster. It is recommended to enable bridged IPv4 traffic to iptables chains when using Flannel. The following command must be run on all Linux nodes:
 
 ```bash
-
 sudo sysctl net.bridge.bridge-nf-call-iptables=1
-
 ```
 
 2. Download & configure Flannel for Linux
 
 Download the most recent Flannel manifest:
 
 ```bash
-
 wget https://raw.githubusercontent.com/coreos/flannel/master/Documentation/kube-flannel.yml
```

Comment:
These will be run on Linux. After modifying it as James suggested, and as it writes above, I think it's pretty self explanatory that it should be run on Linux environment

Reply:
I will also add a Note at the begining, after the "Configuring Flannel..." to tell people that they need to run those commands on a Linux environment

Reply:
Got it. I think it's reasonable to expect that someone adding a Windows node to the scenario would be more comfortable using a Windows PS session. My suggestion here is to add the instructions to run these commands in PS - it's not much of a change anyway and would be greatly appreciated by Windows users.

Reply:
Yes, it would but those are to be run before joining the node to the cluster, so for the moment all they have is their Linux master node. So I am not sure if I shall rewrite them for PS as well.